repos/emacs/update

```diff
@@ -1,61 +1,187 @@
-#!/usr/bin/env nix-shell
-#! nix-shell -i bash -p curl xmlstarlet nix coreutils
-set -euxo pipefail
+#!/usr/bin/env python
```

Comment:
```suggestion
#! /usr/bin/env nix-shell
#! nix-shell ./updater-shell.nix -i python
```

To go along with my suggested rename. This means that the user can still just use `./update` rather than needing to handle entering the nix-shell themselves.

Reply:
This also means the github actions don't need to be updated either.